### PR TITLE
Set helm version to 3.8

### LINKED ISSUE #1217: Set helm version to 3.8

### PR DESCRIPTION
**Pull request by revit13**
Blocked by https://github.com/fybrik/fybrik/pull/1215 as Helm 3.8 is compatible with K8s v1.20-v1.23 according to https://helm.sh/docs/topics/version_skew/

Closes #1217

### CODE REVIEW

File: site/docs/get-started/quickstart.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Many clusters already include cert-manager. Check if `cert-manager` namespace ex
 ```bash
 helm install cert-manager jetstack/cert-manager \
     --namespace cert-manager \
-    --version v1.2.0 \
+    --version v1.6.0 \
```
Comment (ngoracke) [inline site/docs/get-started/quickstart.md:47]: should this be 1.6.2?

File: third_party/cert-manager/Makefile [hunk 2]
```diff
@@ -10,7 +10,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/kubectl create namespace cert-manager
 	$(TOOLBIN)/helm repo add jetstack https://charts.jetstack.io
 	$(TOOLBIN)/helm repo update
-	$(TOOLBIN)/helm install cert-manager jetstack/cert-manager --version v1.2.0 \
+	$(TOOLBIN)/helm install cert-manager jetstack/cert-manager --version v1.6.2 \
```
Comment (ngoracke) [inline third_party/cert-manager/Makefile:13]: This task in pipeline/pipeline.yaml should be updated I think.

  - name: upgrade-helm-certmgr-from-repo
    params:
    - name: helm_repo
      value: https://charts.jetstack.io
    - name: chart_name
      value: jetstack/cert-manager --create-namespace
    - name: release_version
      value: v1.2.0
    - name: release_name
      value: cert-manager
    - name: release_namespace
      value: cert-manager
    - name: overwrite_values
      value: installCRDs=true
    - name: image
      value: $(params.helm_image)
    taskRef:
      kind: Task
      name: helm-upgrade-from-repo
    workspaces:
    - name: source
      workspace: shared-workspace
    when:
    - Input: $(params.deployCertManager)
      Operator: in
      Values:
      - "true"
    - Input: $(params.IMAGE_DIGEST)
      Operator: in
      Values:
      - "none"
    - Input: $(params.INCREMENTAL_RESTART)

File: hack/tools/create_kind.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ op=$1
 
 source ./common.sh
 
-K8S_VERSION=${K8S_VERSION:-v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729}
+K8S_VERSION=${K8S_VERSION:-v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047}
```
Comment (ngoracke) [inline hack/tools/create_kind.sh:11]: out of curiousity, why is this upgrade required?

**Comment by ngoracke** [comment]
Can't view the tekton status currently, (getting 500 errors when clicking on the link), but I'll review that when it's back

File: site/docs/get-started/quickstart.md (hunk 1 above)
Comment (revit13) [reply site/docs/get-started/quickstart.md:47]: Right. this PR does not include documentation changes yet.

**Comment by ngoracke** [comment]
From the latest result, the service policy manager build, opa, manager, katalog, and datacatalog is failing:

```
2022-02-03T09:02:16.9887806Z ++ kubectl get po -l tekton.dev/taskRun=build-and-deploy-run-build-servicepolicymanager-xbwqg --no-headers
2022-02-03T09:02:16.9891637Z ++ cut '-d ' -f1
2022-02-03T09:02:17.0648165Z + kubectl logs build-and-deploy-run-build-servicepolicymanager-xbwqg-pod-bz9hh --all-containers --since=0s
2022-02-03T09:02:17.1539268Z 2022/02/03 09:01:19 Skipping step because a previous step failed
2022-02-03T09:02:17.1579807Z 2022/02/03 09:01:20 Skipping step because a previous step failed
2022-02-03T09:02:17.1646746Z 2022/02/03 08:53:32 Copied /ko-app/entrypoint to /tekton/tools/entrypoint
2022-02-03T09:02:17.1688022Z 2022/02/03 08:53:34 Decoded script /tekton/scripts/script-0-n8m92
2022-02-03T09:02:17.1689359Z 2022/02/03 08:53:34 Decoded script /tekton/scripts/script-1-wphdt
2022-02-03T09:02:17.1690086Z 2022/02/03 08:53:34 Decoded script /tekton/scripts/script-2-5cf4m
2022-02-03T09:02:17.1824440Z + [[ false == \t\r\u\e ]]
2022-02-03T09:02:17.1825274Z + env
2022-02-03T09:02:17.1827426Z EL_TEKTON_TASK_EVENTLISTENER_SERVICE_PORT=8080
2022-02-03T09:02:17.1828337Z LC_MEASUREMENT=en_US.UTF-8
2022-02-03T09:02:17.1829033Z LC_PAPER=en_US.UTF-8
2022-02-03T09:02:17.1829358Z LC_MONETARY=en_US.UTF-8
2022-02-03T09:02:17.1829927Z EL_TEKTON_TASK_EVENTLISTENER_PORT_8080_TCP_PORT=8080
2022-02-03T09:02:17.1830284Z LANG=en_US.UTF-8
2022-02-03T09:02:17.1830879Z HOSTNAME=build-and-deploy-run-build-servicepolicymanager-xbwqg-pod-bz9hh
2022-02-03T09:02:17.1831573Z EL_TEKTON_TASK_EVENTLISTENER_PORT_8080_TCP=tcp://10.96.10.168:8080
2022-02-03T09:02:17.1831979Z EL_TEKTON_TASK_EVENTLISTENER_SERVICE_PORT_HTTP_LISTENER=8080
2022-02-03T09:02:17.1832368Z EL_TEKTON_TASK_EVENTLISTENER_PORT_9000_TCP_PROTO=tcp
2022-02-03T09:02:17.1832768Z EL_TEKTON_TASK_EVENTLISTENER_PORT_9000_TCP_ADDR=10.96.10.168
2022-02-03T09:02:17.1833504Z EL_TEKTON_TASK_EVENTLISTENER_PORT_8080_TCP_PROTO=tcp
2022-02-03T09:02:17.1860368Z EL_TEKTON_TASK_EVENTLISTENER_PORT_8080_TCP_ADDR=10.96.10.168
2022-02-03T09:02:17.1861371Z EL_TEKTON_TASK_EVENTLISTENER_PORT=tcp://10.96.10.168:8080
2022-02-03T09:02:17.1862173Z KUBERNETES_PORT_443_TCP_PROTO=tcp
2022-02-03T09:02:17.1862911Z KUBERNETES_PORT_443_TCP_ADDR=10.96.0.1
2022-02-03T09:02:17.1864409Z EL_TEKTON_TASK_EVENTLISTENER_PORT_9000_TCP=tcp://10.96.10.168:9000
2022-02-03T09:02:17.1864999Z container=oci
2022-02-03T09:02:17.1865788Z LC_NAME=en_US.UTF-8
2022-02-03T09:02:17.1879348Z GO111MODULE=auto
2022-02-03T09:02:17.1879928Z LC_COLLATE=C
2022-02-03T09:02:17.1880570Z GOPATH=/Projects/Proj1
2022-02-03T09:02:17.1881147Z KUBERNETES_PORT=tcp://10.96.0.1:443
2022-02-03T09:02:17.1887261Z ***
2022-02-03T09:02:17.1887526Z HOME=/root
2022-02-03T09:02:17.1887757Z GOROOT=/usr/local/go
2022-02-03T09:02:17.1888161Z LC_CTYPE=en_US.UTF-8
2022-02-03T09:02:17.1888499Z EL_TEKTON_TASK_EVENTLISTENER_PORT_9000_TCP_PORT=9000
2022-02-03T09:02:17.1888947Z KUBERNETES_SERVICE_PORT_HTTPS=443
2022-02-03T09:02:17.1889308Z EL_TEKTON_TASK_EVENTLISTENER_SERVICE_HOST=10.96.10.168
2022-02-03T09:02:17.1889648Z KUBERNETES_PORT_443_TCP_PORT=443
2022-02-03T09:02:17.1890376Z LC_ADDRESS=en_US.UTF-8
2022-02-03T09:02:17.1890697Z LC_NUMERIC=en_US.UTF-8
2022-02-03T09:02:17.1891031Z KUBERNETES_PORT_443_TCP=tcp://10.96.0.1:443
2022-02-03T09:02:17.1891381Z LC_MESSAGES=en_US.UTF-8
2022-02-03T09:02:17.1892160Z EL_TEKTON_TASK_EVENTLISTENER_SERVICE_PORT_HTTP_METRICS=9000
2022-02-03T09:02:17.1892620Z SHLVL=1
2022-02-03T09:02:17.1898520Z KUBERNETES_SERVICE_PORT=443
2022-02-03T09:02:17.1899746Z LC_TELEPHONE=en_US.UTF-8
2022-02-03T09:02:17.1900789Z PATH=/Projects/Proj1/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/SAClientUtil/bin/
2022-02-03T09:02:17.1901494Z LC_IDENTIFICATION=en_US.UTF-8
2022-02-03T09:02:17.1907362Z GOOS=linux
2022-02-03T09:02:17.1907908Z GOCACHE=
2022-02-03T09:02:17.1908713Z GOMODCACHE=
2022-02-03T09:02:17.1909510Z KUBERNETES_SERVICE_HOST=10.96.0.1
2022-02-03T09:02:17.1910029Z GOARCH=amd64
2022-02-03T09:02:17.1911035Z LC_TIME=en_US.UTF-8
2022-02-03T09:02:17.1911448Z _=/usr/bin/env
2022-02-03T09:02:17.1912068Z /workspace/source
2022-02-03T09:02:17.1913685Z + pwd
2022-02-03T09:02:17.1914258Z + cd fybrik/test/services/policymanager
2022-02-03T09:02:17.1916814Z + make source-build
2022-02-03T09:02:17.1924431Z CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o serverpolicymanager serverpolicymanager.go
2022-02-03T09:02:17.1925493Z go: downloading github.com/gin-gonic/gin v1.7.7
2022-02-03T09:02:17.1926764Z go: downloading github.com/rs/zerolog v1.26.0
2022-02-03T09:02:17.1927440Z go: downloading k8s.io/api v0.23.1
2022-02-03T09:02:17.1941289Z go: downloading k8s.io/apimachinery v0.23.1
2022-02-03T09:02:17.1942458Z go: downloading sigs.k8s.io/controller-runtime v0.11.0
2022-02-03T09:02:17.1943290Z go: downloading emperror.dev/errors v0.7.0
2022-02-03T09:02:17.1944073Z go: downloading google.golang.org/grpc v1.43.0
2022-02-03T09:02:17.1946711Z go: downloading github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
2022-02-03T09:02:17.1947519Z go: downloading github.com/xeipuuv/gojsonschema v1.2.0
2022-02-03T09:02:17.1948713Z go: downloading k8s.io/apiextensions-apiserver v0.23.1
2022-02-03T09:02:17.1950074Z go: downloading github.com/gin-contrib/sse v0.1.0
2022-02-03T09:02:17.1950837Z go: downloading github.com/mattn/go-isatty v0.0.14
2022-02-03T09:02:17.1951584Z go: downloading github.com/gogo/protobuf v1.3.2
2022-02-03T09:02:17.1952347Z go: downloading github.com/google/gofuzz v1.2.0
2022-02-03T09:02:17.1953079Z go: downloading k8s.io/klog v1.0.0
2022-02-03T09:02:17.1954155Z go: downloading sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
2022-02-03T09:02:17.1954803Z go: downloading k8s.io/klog/v2 v2.30.0
2022-02-03T09:02:17.1955549Z go: downloading gomodules.xyz/jsonpatch/v2 v2.2.0
2022-02-03T09:02:17.1956634Z go: downloading k8s.io/client-go v0.23.1
2022-02-03T09:02:17.1957733Z go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.2.0
2022-02-03T09:02:17.1958375Z go: downloading github.com/pkg/errors v0.9.1
2022-02-03T09:02:17.1959123Z go: downloading go.uber.org/multierr v1.6.0
2022-02-03T09:02:17.1960168Z go: downloading golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
2022-02-03T09:02:17.1960867Z go: downloading google.golang.org/protobuf v1.27.1
2022-02-03T09:02:17.1962076Z go: downloading golang.org/x/net v0.0.0-20220107192237-5cfca573fb4d
2022-02-03T09:02:17.1980478Z go: downloading sigs.k8s.io/yaml v1.3.0
2022-02-03T09:02:17.1981818Z go: downloading github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415
2022-02-03T09:02:17.2011495Z + set +e
2022-02-03T09:02:17.2011807Z go: downloading k8s.io/apiserver v0.23.1
2022-02-03T09:02:17.2014775Z go: downloading github.com/go-playground/validator/v10 v10.4.1
2022-02-03T09:02:17.2018673Z + kubectl get pipelinerun --no-headers
2022-02-03T09:02:17.2028013Z go: downloading github.com/golang/protobuf v1.5.2
2022-02-03T09:02:17.2028723Z go: downloading github.com/ugorji/go v1.1.7
2022-02-03T09:02:17.2029493Z go: downloading github.com/ugorji/go/codec v1.1.7
2022-02-03T09:02:17.2030247Z go: downloading gopkg.in/yaml.v2 v2.4.0
2022-02-03T09:02:17.2031361Z go: downloading golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
2022-02-03T09:02:17.2032025Z go: downloading gopkg.in/inf.v0 v0.9.1
2022-02-03T09:02:17.2046388Z + grep True
2022-02-03T09:02:17.2047697Z go: downloading github.com/google/go-cmp v0.5.6
2022-02-03T09:02:17.2052805Z go: downloading sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6
2022-02-03T09:02:17.2062503Z go: downloading github.com/go-logr/logr v1.2.0
2022-02-03T09:02:17.2063955Z go: downloading k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
2022-02-03T09:02:17.2113902Z go: downloading github.com/prometheus/client_golang v1.11.0
2022-02-03T09:02:17.2114933Z go: downloading github.com/json-iterator/go v1.1.12
2022-02-03T09:02:17.2116473Z go: downloading github.com/fsnotify/fsnotify v1.5.1
2022-02-03T09:02:17.2117155Z go: downloading go.uber.org/atomic v1.9.0
2022-02-03T09:02:17.2124103Z go: downloading google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368
2022-02-03T09:02:17.2125025Z go: downloading github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb
2022-02-03T09:02:17.2128442Z go: downloading k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
2022-02-03T09:02:17.2130174Z go: downloading github.com/google/cel-go v0.9.0
2022-02-03T09:02:17.2132862Z go: downloading github.com/go-playground/universal-translator v0.18.0
2022-02-03T09:02:17.2135195Z go: downloading github.com/leodido/go-urn v1.2.1
2022-02-03T09:02:17.2137488Z go: downloading golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871
2022-02-03T09:02:17.2139086Z go: downloading go.opentelemetry.io/otel v0.20.0
2022-02-03T09:02:17.2140186Z go: downloading go.opentelemetry.io/otel/trace v0.20.0
2022-02-03T09:02:17.2140858Z go: downloading k8s.io/component-base v0.23.1
2022-02-03T09:02:17.2142197Z go: downloading github.com/evanphx/json-patch v4.12.0+incompatible
2022-02-03T09:02:17.2143967Z go: downloading golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
2022-02-03T09:02:17.2146049Z go: downloading github.com/imdario/mergo v0.3.12
2022-02-03T09:02:17.2147393Z go: downloading github.com/spf13/pflag v1.0.5
2022-02-03T09:02:17.2150973Z go: downloading golang.org/x/term v0.0.0-20210615171337-6886f2dfbf5b
2022-02-03T09:02:17.2152838Z go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
2022-02-03T09:02:17.2153647Z go: downloading github.com/prometheus/client_model v0.2.0
2022-02-03T09:02:17.2156423Z go: downloading github.com/prometheus/common v0.29.0
2022-02-03T09:02:17.2156952Z go: downloading github.com/beorn7/perks v1.0.1
2022-02-03T09:02:17.2157411Z go: downloading github.com/cespare/xxhash v1.1.0
2022-02-03T09:02:17.2157871Z go: downloading github.com/cespare/xxhash/v2 v2.1.2
2022-02-03T09:02:17.2158319Z go: downloading github.com/prometheus/procfs v0.6.0
2022-02-03T09:02:17.2159021Z go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
2022-02-03T09:02:17.2162974Z go: downloading github.com/modern-go/reflect2 v1.0.2
2022-02-03T09:02:17.2163617Z go: downloading github.com/go-openapi/jsonreference v0.19.5
2022-02-03T09:02:17.2164171Z go: downloading github.com/go-openapi/swag v0.19.14
2022-02-03T09:02:17.2164742Z go: downloading github.com/stoewer/go-strcase v1.2.0
2022-02-03T09:02:17.2165380Z go: downloading github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
2022-02-03T09:02:17.2165920Z go: downloading github.com/mitchellh/mapstructure v1.4.3
2022-02-03T09:02:17.2166494Z go: downloading github.com/go-playground/locales v0.14.0
2022-02-03T09:02:17.2168098Z go: downloading sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.25
2022-02-03T09:02:17.2168660Z go: downloading github.com/blang/semver v3.5.1+incompatible
2022-02-03T09:02:17.2169496Z go: downloading go.opentelemetry.io/contrib v0.20.0
2022-02-03T09:02:17.2170397Z go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.20.0
2022-02-03T09:02:17.2172357Z go: downloading go.opentelemetry.io/otel/exporters/otlp v0.20.0
2022-02-03T09:02:17.2173711Z go: downloading go.opentelemetry.io/otel/sdk v0.20.0
2022-02-03T09:02:17.2174362Z go: downloading github.com/davecgh/go-spew v1.1.1
2022-02-03T09:02:17.2176446Z go: downloading github.com/googleapis/gnostic v0.5.5
2022-02-03T09:02:17.2176815Z go: downloading github.com/google/uuid v1.2.0
2022-02-03T09:02:17.2177448Z go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
2022-02-03T09:02:17.2177863Z go: downloading golang.org/x/text v0.3.7
2022-02-03T09:02:17.2178239Z go: downloading github.com/PuerkitoBio/purell v1.1.1
2022-02-03T09:02:17.2178718Z go: downloading github.com/go-openapi/jsonpointer v0.19.5
2022-02-03T09:02:17.2179297Z go: downloading github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210826220005-b48c857c3a0e
2022-02-03T09:02:17.2179713Z go: downloading github.com/mailru/easyjson v0.7.6
2022-02-03T09:02:17.2180070Z go: downloading github.com/felixge/httpsnoop v1.0.1
2022-02-03T09:02:17.2180462Z go: downloading go.opentelemetry.io/otel/metric v0.20.0
2022-02-03T09:02:17.2180882Z go: downloading go.opentelemetry.io/otel/sdk/export/metric v0.20.0
2022-02-03T09:02:17.2181311Z go: downloading go.opentelemetry.io/otel/sdk/metric v0.20.0
2022-02-03T09:02:17.2181936Z go: downloading go.opentelemetry.io/proto/otlp v0.7.0
2022-02-03T09:02:17.2182448Z go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
2022-02-03T09:02:17.2183118Z go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
2022-02-03T09:02:17.2183524Z go: downloading github.com/josharian/intern v1.0.0
2022-02-03T09:02:17.2184029Z go: downloading github.com/grpc-ecosystem/grpc-gateway v1.16.0
2022-02-03T09:02:17.2184422Z # sigs.k8s.io/json/internal/golang/encoding/json
2022-02-03T09:02:17.2185899Z ##[error]/Projects/Proj1/pkg/mod/sigs.k8s.io/json@v0.0.0-20211020170558-c049b76a60c6/internal/golang/encoding/json/encode.go:1249:12: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
2022-02-03T09:02:17.2189497Z ##[error]/Projects/Proj1/pkg/mod/sigs.k8s.io/json@v0.0.0-20211020170558-c049b76a60c6/internal/golang/encoding/json/encode.go:1255:18: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
2022-02-03T09:02:17.2190926Z make: *** [Makefile:14: source-build] Error 2
2022-02-03T09:02:17.3103163Z + rc=1
2022-02-03T09:02:17.3105040Z + [[ 1 -ne 0 ]]
2022-02-03T09:02:17.3105535Z + exit 1
2022-02-03T09:02:17.3120057Z + cleanup
2022-02-03T09:02:17.3121184Z + [[ ! -z /tmp/tmp.1f7LZwQwac ]]
2022-02-03T09:02:17.3122082Z + rm -rf /tmp/tmp.1f7LZwQwac
2022-02-03T09:02:17.3187171Z + echo 'Deleted temp working directory /tmp/tmp.1f7LZwQwac'
2022-02-03T09:02:17.3187686Z Deleted temp working directory /tmp/tmp.1f7LZwQwac
2022-02-03T09:02:17.3189334Z + echo
2022-02-03T09:02:17.3189481Z 
2022-02-03T09:02:17.3212590Z ##[error]Process completed with exit code 1.
```

**Comment by roytman** [comment]
@ngoracke , probably the error is related to https://github.com/kubernetes-sigs/json/issues/8

**Pull request closed without merging**